src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -4969,7 +4969,10 @@ public void testCreateBankAccountStatusTransition_successful() throws Exception
         assertThat(resp, is(equalTo(transitionResponse)));
 
         ArgumentCaptor<HyperwalletStatusTransition> argument = ArgumentCaptor.forClass(HyperwalletStatusTransition.class);
-        Mockito.verify(mockApiClient).post(Mockito.eq("https://api.sandbox.hyperwallet.com/rest/v3/users/test-user-token/bank-accounts/test-bank-account-token/status-transitions"), argument.capture(), Mockito.eq(transition.getClass()));
+        Mockito.verify(mockApiClient).post(Mockito
+                        .eq("https://api.sandbox.hyperwallet.com/rest/v3/users/test-user-token/bank-accounts/test-bank-account-token/status"
+                        + "-transitions"),
```

Comment:
Please make lines 4974 and 4975 into one line and format it.

Reply:
Code formatting is done . Thanks @Rrathinasabapath